### PR TITLE
[ZEND-528] "cmake" addition to required deps

### PR DESCRIPTION
**Pull request by titusen**
CMake is now required to build the project

### CODE REVIEW

**Review by drgora — CHANGES_REQUESTED** [review]
Thanks for the PR. The suggested changes are ok for me, but I believe we should also update the table in `doc/dependencies.md`, and add `cmake` there.

File: doc/dependencies.md [hunk 1]
```diff
@@ -16,3 +16,4 @@ These are the dependencies needed to build Zen on Linux.
 | [Libtool](https://www.gnu.org/software/libtool/) | [2.4.6] |
 | [pkg-config](https://www.freedesktop.org/wiki/Software/pkg-config/) | [0.29.2] |
 | [hexdump](https://www.kernel.org/pub/linux/utils/util-linux/) | [2.37.2] |
+| [cmake](https://cmake.org/download/) | [3.22.1] |
```
Comment (a-petrini) [inline doc/dependencies.md]: If this does not cause any issue or breaks any dependency build, I would lower the version requirement for CMake, as `apt` package manager on Ubuntu Focal by default installs version 3.16.3 (see https://packages.ubuntu.com/focal/devel/cmake). Thanks!

**Review by a-petrini — CHANGES_REQUESTED** [review]

File: doc/dependencies.md (hunk 1 above)
Comment (titusen) [reply doc/dependencies.md]: Done

**Review by drgora — APPROVED** [review]
Looks good to me, thanks

**Review by JackPiri — APPROVED** [review]
The PR is fine to me.
Thanks

**Review by a-petrini — APPROVED** [review]

**Review by ptagl — APPROVED** [review]

**Pull request merged**